starter-code/client/src/app/pages/homepage/homepage.component.html

```diff
@@ -0,0 +1,14 @@
+<p>
+  homepage works!
+</p>
+
+<div>
+  <ul *ngIf = "journals">
+    <li *ngFor = "let journal of journals">
+      {{ journal.title }}
+      {{ journal.date }}
+      <a [routerLink] = "['/journal-entries', journal]">Details</a>
```

Comment:
Care!
On the routerLink you can pass an array on each element will be added to the route on the browser navigator. 
`[routerLink] = "['/journal-entries', journal]"` here you are passing an entire object when you should pass only the `_id`.